examples/pennylane_skip_QGAN.py

```diff
@@ -0,0 +1,382 @@
+"""
+.. _QGAN:
+
+Quantum Generative Adversarial Networks
+=======================================
+
+A QGAN is the quantum version of a *Generative Adversarial Network*.
+These are a particular kind of deep learning model used to generate
+real-looking synthetic data (such as images).
+
+.. figure:: ../../examples/figures/biggan.png
+    :align: center
+    :width: 60%
+    :target: javascript:void(0);
+
+GANs
+^^^^^^
+
+   GANs have a unique structure, consisting of two models:
+
+   -  the **generator**: its goal is to produce realistic-looking data
+      samples.
+
+   -  the **discriminator**: its goal is distinguish fake data produced
+      by the generator from real data.
+
+Training of GANs proceeds as follows:
+
+1. “Real” data is captured in some *training dataset*.
+
+2. The *generator* produces “fake” data by starting with a random input
+   vector and transforming it into an output.
+
+3. The *discriminator* is fed samples of both real and fake data and
+   must decide which label to assign (‘real’ or ‘fake’).
+
+4. Training consists of alternating steps: (i) the generator is frozen
+   and the discriminator is trained to distinguish real from fake. (ii)
+   the discriminator is frozen and the generator is trained to fool the
+   discriminator. The gradient of the discriminator’s output provides a
+   training signal for the generator to improve its fake generated data.
+
+5. Eventually, this training method should converge to a stage where the
+   generator is producing realistic data and the discriminator can’t
+   tell real from fake.
+
+.. note:: 
+    Training is done via the *gradient descent* algorithm,
+    updating only the weights associated with the generator (or vice versa)
+    at each step. There is no internal structure imposed on the generator or
+    discriminator models except that they are differentiable.
+
+.. figure:: ../../examples/figures/gan.png
+    :align: center
+    :width: 90%
+    :target: javascript:void(0);
+
+Qauntum GANs
```

Comment:
That reminds me of my Prof Francesco who ALWAYS makes mistakes in spelling "Quantum", since probably 40 years!

```suggestion
Quantum GANs
```

Reply:
:D this should not have happened! :D